tests/unit/test_auth_merge.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os
+import subprocess
+from pathlib import Path
+
+from deepdiff import DeepDiff
+
+
+ASSETS_ROOT = (Path(__file__).parent.parent / "assets").resolve()
+
+
+# @pytest.mark.skipif(
```

Comment:
Please delete the commented out code if it's not needed

Reply:
configured in a0dbccb4abfaebf